client/src/main/scala/com.gu.contentapi.client/model/MetaResult.scala

```diff
@@ -3,47 +3,36 @@ package com.gu.contentapi.client
 import com.gu.contentapi.client.model.v1._
 import com.gu.contentatom.thrift.Atom
 
-trait MetaResult[Response] {
+trait PaginatedApiResult[Response] {
```

Comment:
Much better 👍 
This makes me wonder whether `PaginatedApiResponse` is more accurate!